test/tests/misc/exception/tests

```diff
@@ -101,4 +101,13 @@
     requirement = 'The system shall support throwing an error during a Jacobian '
                   'calculation, which will terminate the solve.'
   [../]
+
+  [./skip_exception_check]
+    type = 'Exodiff'
+    input = '2d_diffusion_skip_exception.i'
+    exodiff = '2d_diffusion_skip_exception_out.e'
+    expect_out = "Useful information will be ignored if code crash because you skip exception check"
+    allow_warnings = true
+    requirement = "The system should allow users to skip exception check to avoid global communication"
```

Comment:
change "should" to "shall"
change "check" to "checks"
end the requirement with a period.